app/helpers/claims_helper.rb

```diff
@@ -50,4 +50,15 @@ def payment_answers(claim)
   def date_of_birth_string(claim)
     claim.date_of_birth && l(claim.date_of_birth)
   end
+
+  # TODO: This is to be moved to the View Component for the eligible-itt-subject slug/page - not done yet
+  # Temporary helper method for "nqt_in_academic_year_after_itt = false" route
+  def eligible_itt_subjects
```

Comment:
I've got a new class to cover this in my tickets to don't worry about this method too much.